ci/ciimage/android/install.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -e
+
+source /ci/common.sh
+source /ci/env_vars.sh
+
+export DEBIAN_FRONTEND=noninteractive
+export LANG='C.UTF-8'
+
+apt-get -y update
+apt-get -y upgrade
+
+pkgs=(
+  git ninja-build python3-pip sdkmanager
+)
+
+apt-get -y install "${pkgs[@]}"
+
+install_minimal_python_packages
+
+# cleanup
+apt-get -y clean
+apt-get -y autoclean
+
+# sdk install
+
+set -x
+
+if [ -z $ANDROID_HOME -o -z $ANDROID_SDKVER ]; then
+    echo "ANDROID_HOME and ANDROID_SDKVER env var must be set!"
+    exit 1
+fi
+
+test -d ${HOME}/.android || mkdir ${HOME}/.android
```

Comment:
```suggestion
mkdir -p ${HOME}/.android
```

What is the point of testing it exists?